package.json

```diff
@@ -111,22 +111,22 @@
     "react-test-renderer": "16.13.1",
     "regenerator-runtime": "0.13.5",
     "resolve-from": "5.0.0",
-    "rollup": "2.8.2",
-    "rollup-plugin-analyzer": "3.2.2",
+    "rollup": "2.13.1",
+    "rollup-plugin-analyzer": "3.2.3",
     "rollup-plugin-babel": "4.4.0",
     "rollup-plugin-cleanup": "3.1.1",
     "rollup-plugin-delete": "1.2.0",
-    "rollup-plugin-license": "2.0.0",
-    "rollup-plugin-size-snapshot": "0.11.0",
-    "rollup-plugin-typescript2": "0.27.0",
+    "rollup-plugin-license": "2.1.0",
+    "rollup-plugin-size-snapshot": "0.12.0",
+    "rollup-plugin-typescript2": "0.27.1",
     "styled-components": "4.4.1",
-    "stylelint": "13.3.3",
+    "stylelint": "13.6.0",
     "stylelint-config-styled-components": "0.1.1",
-    "stylelint-order": "4.0.0",
+    "stylelint-order": "4.1.0",
     "stylelint-processor-styled-components": "1.10.0",
     "temp": "0.9.1",
-    "ts-jest": "26.0.0",
-    "ts-loader": "7.0.4",
+    "ts-jest": "26.1.0",
+    "ts-loader": "7.0.5",
     "typescript": "3.8.3",
```

Comment:
Note that the typescript `v3.9.3` upgrade was put on hold, and `v3.8.3` is kept. See PR thread comments for more info.